cpp/proto/arcticc/pb2/azure_storage.proto

```diff
@@ -0,0 +1,23 @@
+/*
+Copyright 2023 Man Group Operations Limited
+
+Use of this software is governed by the Business Source License 1.1 included in the file licenses/BSL.txt.
```

Comment:
Minor: BTW, the license file is not actually at this location.

@mehertz We should really fix this. Maybe in Vasil's clang-format PR?